monthly-contributions/src/main/scala/com/gu/support/workers/lambdas/CreatePaymentMethod.scala

```diff
@@ -21,7 +23,8 @@ class CreatePaymentMethod(servicesProvider: ServiceProvider = ServiceProvider)
   def this() = this(ServiceProvider)
 
   override protected def servicesHandler(state: CreatePaymentMethodState, requestInfo: RequestInfo, context: Context, services: Services) = {
-    logger.debug(s"CreatePaymentMethod state: $state")
+    val someTestVariable = "Happy Tuesday, pal!"
+    SafeLogger.error(scrub"IGNORE ME. Leigh-Anne is learning how to scrub logs sent to sentry. Look at this variable I've logged: $someTestVariable")
```

Comment:
I forgot to add exclude the sentry logs in the other appender a la https://github.com/guardian/support-frontend/pull/505 so this log is temporarily here because I want to see that working but otherwise PR comments are very welcome!